readme.md

```diff
@@ -338,6 +338,7 @@ Thanks for contributing! 🦋🙌
 - [](# "prevent-pr-commit-link-loss") [Suggests fixing your PR Commit links before commenting. GitHub has a bug that causes these link to appear as plain commit links, without association to the PR.](https://user-images.githubusercontent.com/1402241/82131169-93fd5180-97d2-11ea-9695-97051c55091f.gif)
 - [](# "align-repository-header") [Aligns the repository header to the repository content on wide screens.](https://user-images.githubusercontent.com/1402241/86574587-587f3800-bf76-11ea-9961-5c25cdb6e357.gif)
 - [](# "linkify-user-edit-history-popup") [Linkifies the username in the edit history popup.](https://user-images.githubusercontent.com/1402241/88917988-9ebb7480-d260-11ea-8690-0a3440f1ebbc.png)
+- [](# "prevent-duplicate-pr-submission") [Disables the create pull request button after submission to prevent creating duplicate pull requests.](https://user-images.githubusercontent.com/16872793/89417135-47a01e80-d6fc-11ea-98fb-724db6647592.gif)
```

Comment:
I think the screenshot gives the right idea, even if not technically correct. The best way would be if you can show the actual clicks in the gif, so you can do a "double click" on the current version.

The description also needs to be updated

Reply:
I can redo it. Should it be disabled or just show that I an clicking 2 times?

Reply:
Yeah it might be a bit confusing. I mean, if clicks can be made visible in the gif, record a new gif showing 2 clicks, without disabling the buttons.

Reply:
I can show clicks

Reply:
Done